src/Task3/Complex.java

```diff
@@ -0,0 +1,58 @@
+package Task3;
+
+public class Complex {
+    private int real;
+    private int imaginary;
+
+    public Complex() {
+        // No need to do this because default values are set to 0
+        this.real = 0;
+        this.imaginary = 0;
+    }
```

Comment:
The comment is absolutely correct, but it is good practice to initialize a value always, since there will be cases when it matters especially with `null` values.

Also you could have called `this(0,0)`